### PR TITLE
feat: PT-5020 Add new controller getCollectionsWithOwnedItems

### PR DESCRIPTION
**Pull request by sanufriev**

### CODE REVIEW

File: openapi.yaml [hunk 1]
```diff
@@ -823,6 +823,56 @@ paths:
         '500':
           $ref: "#/components/responses/ServerError"
 
+
+  /v0.1/ownerships/byCollectionsWithOwnedItems:
```
Comment (dmitry-saraykin) [inline openapi.yaml:827]: I think this should be /v0.1/ownerships/collections?

File: openapi.yaml [hunk 2]
```diff
@@ -823,6 +823,56 @@ paths:
...
+    get:
+      tags:
+        - "NFT Ownerships"
+      summary: "Get collections owned items by owner"
+      description: "Returns list of collection with owned items by specified owner"
+      operationId: getCollectionsWithOwnedItems
+      parameters:
+        - name: "owner"
```
Comment (dmitry-saraykin) [inline openapi.yaml:835]: probably we should be able to accept multiple owners

File: openapi.yaml (hunk 1 above)
Comment (sanufriev) [reply openapi.yaml:827]: np, let do it

File: openapi.yaml (hunk 2 above)
Comment (sanufriev) [reply openapi.yaml:835]: I don't see user case for multiple owners - this controller for acoount page of target user

Comment (sanufriev) [reply openapi.yaml:835]: I think multiple owners is uneeded case that over complicated all

Comment (dmitry-saraykin) [reply openapi.yaml:835]: multiple owners may be required for multi-wallet user accounts

**Review by yakischik — APPROVED** [review]

**Pull request merged**